content/blog/readr-2-1-0-lazy/index.Rmd

```diff
@@ -0,0 +1,132 @@
+---
+output: hugodown::hugo_document
+
+slug: readr-2-1-0-lazy
+title: Lazy reading in readr 2.1.0
```

Comment:
Aren't we back to eager, though?

Reply:
yeah, so you are suggesting we change this to Eager reading in readr 2.1.0?

I wasn't trying to imply 2.1.0 was using lazy, just that the post was about lazy reading in readr 2.1.0. But I can see how it could be read like you suggest.

Reply:
Yeah, I think it should feature "eager" or something about "eager vs lazy reading".

Reply:
I like eager vs lazy reading